tensorflow/lite/kernels/floor_div.cc

```diff
@@ -123,8 +130,21 @@ TfLiteStatus Eval(TfLiteContext* context, TfLiteNode* node) {
       return EvalImpl<int32_t>(context, data->requires_broadcast, input1,
                                input2, output);
     }
+    case kTfLiteFloat32: {
+      return EvalImpl<float>(context, data->requires_broadcast, input1, input2,
+                             output);
+    }
+    case kTfLiteUInt8: {
```

Comment:
For uint8, it also requires changing behavior in the toco toolchain to allow the type to be quantized.